common/darwin/Classes/FlutterWebRTCPlugin.m

```diff
@@ -1764,4 +1764,8 @@ -(NSDictionary*)transceiverToMap:(RTCRtpTransceiver*)transceiver {
     };
     return params;
 }
+
+#pragma mark - RTCPeerConnectionDelegate Methods
+-(void)peerConnection:(RTCPeerConnection*)peerConnection didRemoveIceCandidates:(NSArray<RTCIceCandidate*>*)candidates {}
```

Comment:
I think it would be better to move this line here
https://github.com/flutter-webrtc/flutter-webrtc/blob/main/common/darwin/Classes/FlutterRTCPeerConnection.m#L373